kong/clustering/compat/checkers.lua

```diff
@@ -23,6 +24,47 @@ end
 
 
 local compatible_checkers = {
+  { 3007000000, --[[ 3.7.0.0 ]]
+    function(config_table, dp_version, log_suffix)
+      local has_update
+
+      for _, route in ipairs(config_table.routes or {}) do
+        local snis = route.snis
+        if type(snis) ~= "table" or #snis == 0 then
+          goto continue
+        end
+
+        local idx = 0
+        local new_snis = {}
+        local local_has_update
+        for _, sni in ipairs(snis) do
```

Comment:
This logic only removes the wildcard snis, I am not sure it is the result we expect.
Should we remove the whole route? Could we have more discussion on it?

Reply:
Will remove the compatibility check code as per Datong and Michael's suggestion